testsuite/tests/conftest.py

```diff
@@ -374,10 +375,11 @@ def threescale(testconfig, request):
         tenant = custom_tenant()
         return tenant.admin_api(ssl_verify=testconfig["ssl_verify"], wait=0)
 
-    return client.ThreeScaleClient(
+    return client.ThreeScaleClientCRD(
         testconfig["threescale"]["admin"]["url"],
         testconfig["threescale"]["admin"]["token"],
         ssl_verify=testconfig["ssl_verify"],
+        ocp_provider_ref=testconfig["threescale"]["admin"]["account_ref"],
```

Comment:
I don't know what is `ocp_provider_ref` or `threescale.admin.account_ref` however I am pretty sure `ocp_provider_ref != threescale.admin.account_ref` (scope ot these names ares disjunct). What if they had same name?

Yet "thanks to" `_ref` suffix these names look pretty alien, do not fit to the rest.